tests/test_define.h

```diff
@@ -18,8 +18,8 @@ extern "C" {
   "RVORZ9SIIP9RCYMREUIXXVPQIPHVCNPQ9HZWYKFWYWZRE9JQKG9REPKIASHUUECPSQO9JT9XNM" \
   "VKWYGVA"
 #define TRYTES_81_3                                                            \
-  "FPPBU9AOXZHSJEWDCPWGCPKGOVIUWWPDTIIXDJUKYYNNPTWCGDWFBZLKVCLOKGYEHJUMABSTA9" \
-  "OP99999"
+  "UVINHLDECHCNTYOMQZTVCQVHNVNACDNIQYUV9KYNDUMHMTDAQGPIBIDZD9XZEDDYPSRHYAU9DM" \
+  "EGA9999"
```

Comment:
I wonder if this situation will happen again in the future.

Reply:
yes. we need a solution.